saagieapi/jobs/gql_queries.py

```diff
@@ -230,18 +294,26 @@
     job(id: $jobId){
         id
         name
+        alias
         description
         alerting{
             emails
             loginEmails{
-              login
-              email
+                login
+                email
             }
             statusList
         }
         instances(limit: $instancesLimit){
             id
             status
+            history {
```

Comment:
The doc of the section `Getting job information` should be updated (with field `history`)